test/aes_nangate45.metrics_limits

```diff
@@ -1,23 +1,23 @@
 {
   "IFP::instance_count" : "20110.8"
- ,"DPL::design_area" : "37108.799999999996"
- ,"DPL::utilization" : "4.08"
- ,"RSZ::repair_design_buffer_count" : "415"
+ ,"DPL::design_area" : "37658.4"
+ ,"DPL::utilization" : "4.2"
+ ,"RSZ::repair_design_buffer_count" : "411"
  ,"RSZ::max_slew_slack" : "0"
  ,"RSZ::max_capacitance_slack" : "0"
  ,"RSZ::max_fanout_slack" : "0"
- ,"RSZ::worst_slack_min" : "-0.07986893504961975"
- ,"RSZ::worst_slack_max" : "-0.31123768520158157"
- ,"RSZ::tns_max" : "-166.5066995815757"
- ,"RSZ::hold_buffer_count" : "97"
+ ,"RSZ::worst_slack_min" : "-0.0802665198037703"
+ ,"RSZ::worst_slack_max" : "-0.2756862335238735"
+ ,"RSZ::tns_max" : "-160.4952414135037"
+ ,"RSZ::hold_buffer_count" : "158"
```

Comment:
This is kind of an interesting jump. Any thoughts on the root cause?

Reply:
This is due to auto parameter inferencing for cluster diameter and cluster size in CTS.   clock_tree_synthesis no longer requires -sink_clustering_max_diameter or -sink_clustering_size arguments to be specified accurately per design.  Setup TNS improved from -175.3 to -169.3, setup WNS from -0.133 to -0.097 even though hold buffer count increased from 97 to 158.